README.md

```diff
@@ -162,6 +167,52 @@ the generic documentation for more information on the process.
    $ kubectl scale --replicas=1 -n kube-system deployment/sealed-secrets-controller
    ```
 
+### Migration from SealedSecret <0.8.0 to 0.8.0+
+
+In versions prior to 0.8.0 the controller was installed into
+`kube-system` namespace by default but to support managed services like
+GKE the default was changed to `sealed-secrets`.
+[Issue ref](https://github.com/bitnami-labs/sealed-secrets/issues/90)
+
+To migrate, follow this procedure:
+
+1. Backup your existing encryption private key:
+   ```sh
+   $ kubectl get secret -n sealed-secrets sealed-secrets-key -o yaml >master.key
+   ```
+
+2. Prepare to load your encryption private key into the new namespace:
+   ```sh
+   $ sed -i 's/kube-system/sealed-secrets/g' master.key
+   ```
+   Or on Mac OS:
+   ```sh
+   $ sed -i '' 's/kube-system/sealed-secrets/g' master.key
+   ```
+
```

Comment:
I think we should create the secret here, before installing the new controller.  That way we don't need to restart the controller post-install.

Something like:
```markdown
3. Load the existing master secret key into the new namespace:
    ```sh
    $ kubectl create -f moved.key
    ```
```

What I _actually_ think we should do is combine these 3 steps into one, since I think it's easier to see what's going on as a single step (please disagree if you don't think so):

```markdown
1. Copy your existing master key to the new namespace:
    ```sh
    $ kubectl create namespace sealed-secrets
    $ kubectl get secret -n sealed-secrets sealed-secrets-key -o yaml |
       sed 's/kube-system/sealed-secrets/' |
       kubectl create -f -
    ```
2. Install the new controller.
3. Delete the old kube-system controller.
4. Verify unsealing new/modified keys is working as expected, then delete the old kube-system master key.
```

Reply:
I've gone ahead and updated this, can always re-order 3+4 if there's any possibility for race conditions.

Reply:
We already have the possibility of 2 controllers running simultaneously for a brief period during a "normal" controller upgrade (same namespace/Deployment upgrade).  This should be fine (at worst we decrypt a SealedSecret twice).